source/stylesheets/components/_index.scss

```diff
@@ -0,0 +1,61 @@
+.index {
+    padding-top: $spacing-small;
+    text-align: center;
+}
+
+.index__cards {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    padding: $spacing-base;
+    position: relative;
+
+    @include breakpoint($breakpoint-medium) {
+        padding: $spacing-large;
+    }
+
+    @include breakpoint($breakpoint-large) {
+        flex-direction: row;
+    }
+}
+
+.index__case {
+    position: relative;
+}
+
+.index__case--overlay {
+    background-color: $teal-transparent;
+    bottom: 0;
+    color: $white;
+    height: 100%;
+    left: 0;
+    opacity: 0;
+    position: absolute;
+    right: 0;
+    top: 0;
+    width: 100%;
```

Comment:
You don't need this `width` declaration, since you are declaring it expand to `left: 0` and `right: 0`